packages/quantic/force-app/examples/main/lwc/actionPerformRecommendationSearch/actionPerformRecommendationSearch.html

```diff
@@ -0,0 +1,11 @@
+<template>
+  <div class="slds-grid slds-gutters">
+    <lightning-button
+      label="Perform recommendation search"
```

Comment:
This needs to be updated, we don't use the term recommendation search.
```suggestion
      label="Get recommendations"
```